contracts/contracts/rateX/RateX.sol

```diff
@@ -1,222 +1,235 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.0;
+pragma solidity 0.8.24;
 
-import "./interfaces/IDex.sol";
-import "./libraries/TransferHelper.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import {IDex} from './interfaces/IDex.sol';
+import {TransferHelper, IERC20} from './libraries/TransferHelper.sol';
+import {Ownable2Step} from '@openzeppelin/contracts/access/Ownable2Step.sol';
```

Comment:
This is highly opinionated but If we are using only one simple contract from openzeppelin, adding dependency is overhead in this case